Description: Fix test_request_cookie_parsing()
 This patch cherry-picks change in this function from upstream HEAD.
Author: Thomas Goirand <zigo@debian.org>
Bug-Debian: https://bugs.debian.org/811490
Forwarded: not-needed
Last-Update: 2016-01-20

--- python-falcon-0.3.0.orig/tests/test_cookies.py
+++ python-falcon-0.3.0/tests/test_cookies.py
@@ -131,7 +131,7 @@ class TestCookies(testing.TestBase):
     def test_request_cookie_parsing(self):
         # testing with a github-ish set of cookies
         headers = [
-            ('Cookie', '''Cookie:
+            ('Cookie', '''
                 logged_in=no;_gh_sess=eyJzZXXzaW9uX2lkIjoiN2;
                 tz=Europe/Berlin; _ga=GA1.2.332347814.1422308165;
                 _gat=1;
